tests/lcs/core/interface/image-undo.livecodescript

```diff
@@ -0,0 +1,58 @@
+script "CoreEngineClipboard"
```

Comment:
Change the name of the test...

Also need other tests for cases such as getting the text in undoChanged, setting the text, setting the filename, setting the imageData etc. - i.e. things which might cause the new code to be triggered.